.github/workflows/run.yml

```diff
@@ -0,0 +1,71 @@
+name: Run
+on:
+  schedule:
+    - cron: |
+        0 0 * * *
+  pull_request:
```

Comment:
We should remove this on merge, right?

Reply:
Yes.
I'll remove this definition before merge.
Currently, I remain this definition for debugging.

Reply:
I've removed this in https://github.com/pgroonga/docker/pull/18/commits/48eadf308dd73217c443bba67a1875fc7c3793b2.